packages/ui/src/context/router-context.tsx

```diff
@@ -1,12 +1,12 @@
 import { ComponentType, createContext, ReactNode, useContext } from 'react'
-import type {
-  LinkProps,
-  Location,
-  NavigateFunction,
-  NavLinkProps,
-  OutletProps,
-  Params,
-  UIMatch
+import {
+  type LinkProps,
+  type Location,
+  type NavigateFunction,
+  type NavLinkProps,
+  type OutletProps,
+  type Params,
+  type UIMatch
```

Comment:
Curious why we needed this change?

Reply:
No need - I was testing the eslint ban rule and just forgot to roll this back